tasks/test.js

```diff
@@ -59,7 +59,8 @@ module.exports = function (grunt) {
 
   grunt.registerTask('test:quick', [
     'test:server',
-    'test:ui',
+    'checkPlugins',
```

Comment:
Might as well move `checkPlugins` to the top of the task, since it's advisory and when it's buried under the output from `test:server` it is harder to notice.